PostalRateCalculator-Backend/src/main/java/PostalRateCalculator/service/EnvelopeService.java

```diff
@@ -65,24 +65,24 @@ private static void weightRangeCheck(double weight, WeightUnit weightUnit) {
 
 
 
-    @Transactional
-    public Envelope createEnvelope(String widthString, String lengthString, String weightString, String sizeUnitString, String weightUnitString) {
-        double width = Double.parseDouble(widthString);
-        double length = Double.parseDouble(lengthString);
-        double weight = Double.parseDouble(weightString);
-
-        SizeUnit sizeUnit = null;
-        WeightUnit weightUnit = null;
-
-        if (sizeUnitString.equals("mm")) { sizeUnit = SizeUnit.Millimeters; }
-        if (sizeUnitString.equals("in")) { sizeUnit = SizeUnit.Inches; }
-
-        if (weightUnitString.equals("g")) { weightUnit = WeightUnit.Grams; }
-        if (weightUnitString.equals("oz")) { weightUnit = WeightUnit.Ounces; }
-
-        Envelope envelope = new Envelope(width, length, weight, sizeUnit, weightUnit);
-        envelope = envelopeRepository.save(envelope);
-
-        return envelope;
-    }
+    //@Transactional
+//    private Envelope createEnvelope(String widthString, String lengthString, String weightString, String sizeUnitString, String weightUnitString) {
+//        double width = Double.parseDouble(widthString);
+//        double length = Double.parseDouble(lengthString);
+//        double weight = Double.parseDouble(weightString);
+//
+//        SizeUnit sizeUnit = null;
+//        WeightUnit weightUnit = null;
+//
+//        if (sizeUnitString.equals("mm")) { sizeUnit = SizeUnit.Millimeters; }
+//        if (sizeUnitString.equals("in")) { sizeUnit = SizeUnit.Inches; }
+//
+//        if (weightUnitString.equals("g")) { weightUnit = WeightUnit.Grams; }
+//        if (weightUnitString.equals("oz")) { weightUnit = WeightUnit.Ounces; }
+//
+//        Envelope envelope = new Envelope(width, length, weight, sizeUnit, weightUnit);
+//        envelope = envelopeRepository.save(envelope);
+//
+//        return envelope;
+//    }
```

Comment:
so... nothing is happening here?

Reply:
that's gonna be removed at the end
I just used that to test that i could add envelopes to the database